jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -611,11 +624,11 @@ public int getMaxReservedThreads()
     }
 
     /**
-     * @return the number of available reserved threads
+     * @return the number of available reserved threads that are currently reserved
```

Comment:
The word `currently` could be assumed implicitly given its natural meaning in a typical sentence, i.e. in this case there is probably some different meaning that could be phrased more explicitly